core/modules/field/field.install

```diff
@@ -93,3 +93,24 @@ function field_update_1001() {
     $config->save();
   }
 }
+
+/**
+ * Grant the new "administer fields" permission to trusted users.
+ */
+function field_update_1002() {
+  // Assign the permission to anyone that already has a trusted core permission
+  // that would have previously let them administer fields on an entity type.
+  $rids = array();
+  $permissions = array(
+    'administer site configuration',
+    'administer content types',
+    'administer users',
+  );
+  foreach ($permissions as $permission) {
+    $rids = array_merge($rids, array_keys(user_roles(FALSE, $permission)));
+  }
+  $rids = array_unique($rids);
+  foreach ($rids as $rid) {
+    _update_7000_user_role_grant_permissions($rid, array('administer fields'), 'field');
```

Comment:
We don't have this function, we'll need to load the config files and change them manually.